code/src/components/subintro/Subintro.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import Subintro from "@/components/subintro/Subintro";
+import  subintroData  from "@/components/subintro/Subintro.constants";
+jest.mock("@/assets/images/subintro/moped-icon.jpg", () => { });
+jest.mock("@/assets/images/subintro/box-icon.jpg", () => { });
+jest.mock("@/assets/images/subintro/percent-icon.jpg", () => { });
+jest.mock("@/assets/images/subintro/clock-icon.jpg", () => { });
```

Comment:
Maybe we need to wait until #45 will be merged. Then we should remove these lines

Reply:
Yes, i also think it will be better to wait. I will remove those mocks.